test/unit/liquid/filters/rails_helpers_test.rb

```diff
@@ -10,7 +10,7 @@ def setup
 
   test 'javascript_include_tag' do
     @context.registers[:controller] = ApplicationController.new
-    Webpacker.manifest.stubs(:lookup_pack_with_chunks!).with('stats', type: :javascript).returns('/packs/stats.js')
+    WebpackHelper.expects(:javascript_packs_with_chunks_tag).returns('/packs/stats.js') # TODO: FIXME!!!
```

Comment:
What requires to be fixed here?

Reply:
Since WebpackHelper is a module I can't find a way to mock `javascript_packs_with_chunks_tag`.